lib/active_model/serializer/adapter/flatten_json.rb

```diff
@@ -6,6 +6,13 @@ def serializable_hash(options = {})
           super
           @result
         end
+
+        private
+
+        # no-op
```

Comment:
`no-op: The FlattenJson adapter does not include meta data`? `no-op: The FlattenJson adapter only serializes attributes`?